static/sass/styles.scss

```diff
@@ -8,15 +8,19 @@ $color-tabs-active-bar: #e95420;
 $color-brand: #7c355d;
 $color-dark: #262626;
 $color-paper: #f3f3f3;
+
+//XXX: move this to vanilla
+$border-radius: 0;
+
 $nudge--small: 0.55rem;
 $p-small-lh-diff: map-get($line-heights, default-text) -
   map-get($line-heights, small);
 
 @import "pattern_animations";
 @import "pattern_application";
 @import "pattern_buttons";
-@import "pattern_blog-card";
-@import "pattern_cards";
+// @import "pattern_blog-card";
+// @import "pattern_cards";
```

Comment:
Can you remove these lines

Reply:
uncommented them, we agreed to clean up at the end to avoid needless rebases